documentation/docs/troubleshooting.md

```diff
@@ -8,16 +8,19 @@ keywords:
 - solution
 - discord
 - stackexchange
+- reference
 ---
 # Troubleshooting
 
-
 ## StackExchange
 
-> [https://iota.stackexchange.com](https://iota.stackexchange.com/)
+The [IOTA StackExchange](https://iota.stackexchange.com/)
+is an excellent tool for developers to find answers to their problems. Just search for your problem and find your
```

Comment:
Rather subjective, no?

Reply:
how so?

Reply:
"Excellent" is a bold statement to make. It basically means that you can find a full answer to your problem every time. As for medium, this is not true, because some information you can find not on a SE, but in discord chats, working chats, wikis, documentation, youtube videos, textbooks. In comparison to other state exchanges, I assume that our stack exchange has about as much content as any other stack exchange.

Also "Just search" is not quite easy when you have a rare problem or a problem that is hard to formalize into a search query, so I would avoid [evaluating the ease of use](https://wiki.iota.org/participate/contribute-to-wiki/for_tws/form#be-human) here as well.

Either way, all of this is minor and doesn't deserve an outblown discussion. I'll approve whenever you decide to fix it or not.